### PR TITLE
Adds srcset question to HTML

### PR DESCRIPTION
**Pull request by jhummel**

### CODE REVIEW

**Comment by stepek** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ This file contains a number of front-end interview questions that can be used wh
 * Describe the difference between `<script>`, `<script async>` and `<script defer>`.
 * Why is it generally a good idea to position CSS `<link>`s between `<head></head>` and JS `<script>`s just before `</body>`? Do you know any exceptions?
 * What is progressive rendering?
+* Why you would use a srcset attribute in an image tag? Explain the process the browser uses when evaluating the content of this attribute.
```
Comment (cezaraugusto) [inline README.md]: this looks nice, sorry this took so long. 

could you please add a backtick to srcset for better readability? markdown will then render it in a nice gray :) `srcset` > srcset

```diff
- Why you would use a srcset attribute in an image
+ Why you would use a `srcset` attribute in an image
```

Comment (jhummel) [reply README.md]: @cezaraugusto done.

**Comment by roblarsen** [comment]
Thanks!

**Pull request merged**